HotelYeah/src/index.tsx

```diff
@@ -2,11 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import store from './store/configureStore';
+import { Provider } from 'react-redux';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <React.StrictMode>
+    <Provider store={store}>
```

Comment:
this may be an indicator that you hadn't pulled the 'remove redux' changes from main, let's see if we can merge those into your branch before merging your changes